doc/api.yml

```diff
@@ -2044,7 +2054,30 @@ paths:
           $ref: '#/components/responses/429'
         '500':
           $ref: '#/components/responses/500'
-
+  /fees/stats:
+    get:
+      tags:
+        - Paying
+      summary: Get monthly statistics on Fees
+      parameters:
+        - name: start_month
+          in: query
+          schema:
+            type: string
+            format: date
+        - name: end_month
```

Comment:
same here use the month_from and month_to, add "the actual month by default" in your description. We will return the stats of the month by default if both of them are not provided